guides/plugins/apps/storefront/cookies-with-apps.md

```diff
@@ -103,6 +296,61 @@ If you are not familiar with setting up Storefront snippets, please refer to our
 
 <PageRef page="../../plugins/storefront/add-translations" />
 
+## Automatic Configuration Change Detection
```

Comment:
should also be clear with the concept? do we really need a that detailed description, when the hash changes for an app?

Reply:
does not hurt to tell it again in that place, to be honest